project.clj

```diff
@@ -32,19 +28,19 @@
                  [reagent "0.8.1"]
                  [soda-ash "0.76.0"]
 
+                 [cljs-web3 "0.19.0-0-11"]                  ; needed ?
```

Comment:
No, it shouldn't because https://github.com/district0x/d0x-libs/blob/master/shared/cljs-web3-next/src/deps.cljs#L2 will get it